roles/docker_node/tasks/main.yml

```diff
@@ -3,12 +3,19 @@
     name: docker-ce
```

Comment:
@micahkemp-splunk  let's make this a variable with a default value of docker-ce.

Reply:
Approving since it'll be in the next PR